roles/neutron-data-network/tasks/monitoring.yml

```diff
@@ -28,3 +28,14 @@
 - name: l3 router check
   sensu_check: name=check-l3-router plugin=check-l3-router.sh use_sudo=true
   notify: restart sensu-client
+
+- name: default route gateway
+  shell: ip route | awk "/default.+{{ hostvars[inventory_hostname][primary_interface].device }}/ { print \$3; exit }"
```

Comment:
Does the route information not exist in the Ansible generated facts?

This could use some comments that explain the expected input and output of the awk, for future maintenance.